master/buildbot/status/buildstep.py

```diff
@@ -380,27 +380,28 @@ def upgradeToVersion5(self):
         self.wasUpgraded = True
 
     @defer.inlineCallbacks
-    def prepare_trigger_links(self):
+    def prepare_trigger_links(self, codebases_arg):
```

Comment:
minor: It would be nice to add a docstring

Reply:
Done